### PR TITLE
chore(upgrade): add known issue in report

### PR DESCRIPTION
**Pull request by xzf0587**
add `simpleAuthEndpoint in configuration is invalid` known issue.
image

### CODE REVIEW

File: packages/fx-core/templates/core/v3Migration/upgradeReport.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ For more detailed changes, please refer to this [wiki](https://aka.ms/teams-tool
 
 2. If your project is created with Viusal Studio version < 17.4, you may see error `InvalidParameter: Following parameter is missing or invalid for aadApp/create action: name` when executing commands. Please follow [the steps](#how-to-roll-back) to roll back, install VS 17.4 and run upgrade first.
 
+3. If your tab app is created with Teams Toolkit 3.2.0 or earlier version, you may see error `simpleAuthEndpoint in configuration is invalid` when local debugging your app. Please follow this [wiki]( `https://aka.ms/teams-toolkit-5.0-upgrade#simpleauthendpoint-in-configuration-is-invalid`) to learn how to mitigate this error.
```
Comment (blackchoey) [inline packages/fx-core/templates/core/v3Migration/upgradeReport.md:40]: remote preview should also have same issue.

File: packages/fx-core/templates/core/v3Migration/upgradeReport.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ For more detailed changes, please refer to this [wiki](https://aka.ms/teams-tool
 
 2. If your project is created with Viusal Studio version < 17.4, you may see error `InvalidParameter: Following parameter is missing or invalid for aadApp/create action: name` when executing commands. Please follow [the steps](#how-to-roll-back) to roll back, install VS 17.4 and run upgrade first.
```
Comment (blackchoey) [inline packages/fx-core/templates/core/v3Migration/upgradeReport.md:38]: Not blocking this PR. @KennethBWSong is this known issue still valid?

Comment (KennethBWSong) [reply packages/fx-core/templates/core/v3Migration/upgradeReport.md:38]: I think it is still valid as long as we don't have any update on migration of projects before 17.4.

File: packages/fx-core/templates/core/v3Migration/upgradeReport.md (hunk 1 above)
Comment (xzf0587) [reply packages/fx-core/templates/core/v3Migration/upgradeReport.md]: Fixed. 
The remote debugging will trigger this error.

**Review by blackchoey — APPROVED** [review]

**Pull request merged**